### PR TITLE
fix findings in README file

### PR DESCRIPTION
**Pull request by ngoan1608**
Hi Holger,
Hi Thomas,

This PR contains changes for findings in README files as issue #98 .
- I have updated as per suggestions from Holger.
- The build's status badge ([Build RobotFramework AIO packages](https://github.com/test-fullautomation/RobotFramework_AIO/actions/workflows/build_robotframework_aio.yml) contains "failing" which is generated by Github bases on build's status) is moved to the top of README file. In case it make confused for readers, please let me know, I will remove it.

Please review and give your idea about the changes.

Thank you,
Ngoan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,37 +82,38 @@ Then follow below steps for building process:
 	./build
 	```
 
-Build the RobotFramework AIO package with all related libraries (defined in `config/repositories/repositories.conf` file).
+Build the RobotFramework AIO package with all related libraries 
+(defined in `config/repositories/repositories.conf` file).
 
-Build script will detect the Operation System (**Windows** or **Linux**) automatically to run the appropriate steps for building installer package.
+Build script will detect the Operation System (**Windows** or **Linux**)
```
Comment (HolQue) [inline README.md]: I would prefer "operating system" (instead of "Operation System").

**Review by HolQue — COMMENTED** [review]
Fine to me now!

File: README.md (hunk 1 above)
Comment (ngoan1608) [reply README.md]: Hi Holger,

Thank you for your finding.
I have corrected it in the new commit.

Thank you,
Ngoan

**Review by test-fullautomation — APPROVED** [review]
Hi Ngoan,

looks good to me!

Thank you,
 Thomas

**Pull request merged**